src/components/AnchorButton.js

```diff
@@ -0,0 +1,24 @@
+import styled, { css } from 'styled-components'
+
+const HoverSyles = css`
+  background-color: #0062cc;
```

Comment:
also consider using `theme` with `styled-components`, it uses React context api under the hood to be able to add theming support for your app.